client/src/main/webjar/app.js

```diff
@@ -8,49 +8,158 @@
         ]
     );
 
-    function User(
+    function ExampleController(
```

Comment:
All of the code in app.js and index.html is just an example of using the new endpoints.

@mckoon this is all throwaway code and you can ignore it and just delete the file if your work gets in first.